crates/vm/levm/src/vm.rs

```diff
@@ -229,7 +229,7 @@ impl VM {
         }
     }
 
-    pub fn execute(
+    pub fn run_execute(
```

Comment:
IIRC we agreed on `run_execution`
```suggestion
    pub fn run_execute(
```

Reply:
Oh sorry, my bad. I'll change it right away. 
Quick question however, should we change the name of the return type as well?

Now the `execute` function returns a _Transaction_Report. Should the return value be renamed to match? 

Something like `ExecutionReport`?

Reply:
Yes! I agree.

Reply:
Credit where credit is due: This was @tomip01's idea.

Reply:
While I was at it, I also rename `TxResult` to `ExecutionResult`.

What do you think Ivan?

Reply:
Let me know when this is addressed.

Reply:
Done here: https://github.com/lambdaclass/ethrex/pull/1850/commits/5210a7c864c1308e3743313c24c77dfdbf9d4655